docs/error-reporting/advanced/sync-submissions.md

```diff
@@ -0,0 +1,24 @@
+---
+id: sync-submissions
+title: Performing Synchronous Submissions
+sidebar_label: Synchronous Submissions
+description: Perform followup operations based on the result of a submission.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+For performance reasons, error submissions are normally handled asynchronously from the HTTP client on the object store server. However, there are cases where it can be useful to perform follow-up operations based on the result of a submission or directly receive errors found during submission processing. This article demonstrates how to use the synchronous submission facility.
```

Comment:
```suggestion
For performance reasons, error submissions are normally handled asynchronously from the HTTP client on the object store server. However, there are cases where it can be valuable to perform follow-up operations based on the result of a submission or directly receive errors found during submission processing. This article demonstrates how to use the synchronous submission facility.
```